app/tests/test_blast.py

```diff
@@ -9,7 +9,7 @@ def test_read_config():
 	with TestClient(app) as client: #include @startup hook
 		with open('/data/blast_config.json') as f:
 			config = json.load(f)
-		response = client.get('/blast/config')
+		response = client.get('/api/tools/blast/config')
```

Comment:
I think we should use `API_PREFIX` everywhere instead of hardcoding.

Reply:
@veidenberg  Done